meteor-app/imports/ui/member/member-card-small.js

```diff
@@ -10,7 +10,7 @@ const MemberCardSmall = (props) => {
   return (
     <List.Item
       style={{ padding: '20px' }}
-      onClick={props.onCardClick ? props.onCardClick.bind(null, _id) : null}
+      onClick={() => props.onCardClick(_id)}
```

Comment:
When I ran this branch, I was still get failures around onCardClick not being a function. It may not be just here that the problem is - something to hunt down